doc/reference/reference_lua/datetime.rst

```diff
@@ -1012,3 +1036,148 @@ Support for relational operators for ``interval`` objects has been added since :
     ---
     - true
     ...
+
+Leap second
+-----------
+
+Coordinated Universal Time (UTC) is based on International Atomic Time (TAI).
+Atomic clocks are more stable than the rate of the earth rotation since the
+latter undergoes a full range of geophysical perturbations at various time
+scales. Leap seconds are needed to keep the two time scales in agreement,
+i.e. UT1-UTC smaller than 0.9 second. Therefore, when necessary a "leap second"
+is applied to UTC. Since the adoption of this system in 1972 it has been
+necessary to add a number of seconds to UTC. It is theorically possible to have
+a negative leap second (a second removed from UTC), but so far,
+all leap seconds have been positive (a second has been added to UTC). Based
+on what we know about the earth's rotation, it is unlikely that we will ever
+have a negative leap second.
+
+The standard way to count UTC time across a leap second is:
+
+..  code-block:: tarantoolsession
+
+    2011-12-31 23:59:57
+    2011-12-31 23:59:58
+    2011-12-31 23:59:59
+    2011-12-31 23:59:60 <-- leap second
+    2012-01-01 00:00:00
+    2012-01-01 00:00:01
+    2012-01-01 00:00:02
+
+The decision to insert a leap second into UTC is currently the responsibility
+of the International Earth Rotation and Reference Systems Service. The IERS is
+measuring the true earth rotation and determines when a leap second has to be
+inserted. Insertion of a leap second is always scheduled for the end of the
+last day of a month, preferably at the end of June or December, at UTC midnight.
+Announcements whether a leap second is scheduled or not are published by the
+IERS in their `Bulletin C <https://hpiers.obspm.fr/iers/bul/bulc/bulletinc.dat>`__.
+
+`IANA <https://www.iana.org/time-zones>`__ (Internet Assigned Numbers Authority)
+maintains a so-called Olson TZ database and provides a set of time zone
+description files, including daylight saving information for the current year
+as well as for past years. Beside the time zone description files it also
+contains a leapseconds file. The TZDB version of the file is used to
+implement the time zones support properly.
+
+This section describes how ``datetime`` module supports leap seconds:
+
+*   Tarantool includes TZDB (also called ``tzdata``) and uses this database
+    for a correct time zones support. Use Lua module :doc:`/reference/reference_lua/tarantool`
+    to get a used version of ``tzdata``:
+
+    Since: :doc:`3.2.0 </release/3.2.0>`
```

Comment:
`Since` between the description and example looks weird. Ideally, we should describe all the fields in the [Module tarantool](https://docs.d.tarantool.io/en/doc/gh-xxxx-missed-datetime-information/reference/reference_lua/tarantool/) docs and add the `Since` info there. As a workaround, let's update the previous sentence like this:

```
Since :doc:`3.2.0 </release/3.2.0>`, you can use the Lua module ....
```

Reply:
Updated:

```diff
--- a/doc/reference/reference_lua/datetime.rst
+++ b/doc/reference/reference_lua/datetime.rst
@@ -1082,10 +1082,9 @@ implement the time zones support properly.
 This section describes how ``datetime`` module supports leap seconds:
 
 *   Tarantool includes TZDB (also called ``tzdata``) and uses this database
-    for a correct time zones support. Use the Lua module :ref:`tarantool <tarantool-module>`
-    to get a used version of ``tzdata``:
-
-    Since: :doc:`3.2.0 </release/3.2.0>`
+    for a correct time zones support. Since :doc:`3.2.0 </release/3.2.0>`,
+    you can use the Lua module :ref:`tarantool <tarantool-module>` to get
+    a used version of ``tzdata``:
 
 ..  code-block:: tarantoolsession
 
```

but I've removed this paragraph for now, I'll add it in scope of doc request from #10387.